cypress/support/fragments/inventory/inventorySearch.js

```diff
@@ -151,5 +151,13 @@ export default {
   },
   switchToHoldings: () => {
     cy.do(Button({ id: 'segment-navigation-holdings' }).click());
+  },
+
+  instanceTabIsDefault() {
+    cy.do(
+      Button({ id: 'segment-navigation-instances' }).perform(element => {
```

Comment:
are you sure that it can't be implemented through including()/matching()? There is example from frontside:
Link({
  href: and(
    matching(/^https/),
    including('google'),
    matching(/\.com$/)
  )
}).exists();

Reply:
Default instance has a class "primary". With your suggestions I don't see how I can check class. May be I'm wrong